libs/libcommon/src/libcommon/config.py

```diff
@@ -109,6 +109,10 @@ class ProcessingGraphConfig:
             "/dataset-info": {"input_type": "dataset", "requires": "/parquet-and-dataset-info"},
             "/split-names-from-dataset-info": {"input_type": "config", "requires": "/dataset-info"},
             "/sizes": {"input_type": "dataset", "requires": "/parquet-and-dataset-info"},
+            "dataset--split-names": {
+                "input_type": "dataset",
+                "requires": "/split-names-from-streaming",
```

Comment:
I thought the same but then I realized we dont support having multiple parents for a processingStep.
Do you think is a good moment to add support for it? I was thinking instead, having two different steps:
- `dataset--split-names-from-streaming` requires `/split-names-from-streaming`
- `dataset--split-names-from-dataset-info` requires` /split-names-from-dataset-info`
And for the `/splits?dataset=...` endpoint, to have 
```
"/splits": {
                "dataset": ["/splits", "dataset--split-names-from-streaming", "dataset--split-names-from-dataset-info],
                "config": ["/split-names-from-streaming", "/split-names-from-dataset-info"],
            },
```
What do you think? 
In case of adding multiple parents, I am afraid about the datasets that has a lot of configs e.g if a dataset has 1000 configs, the processing step "`dataset--split-names`" will be called 2000 times. It will take probably same times as if we have two different steps, but at least, we could have the response early but not sure.

Reply:
Will close this PR and send separated ones for dataset-split-names-from-streaming and dataset-split-names-from-dataset-info